src/server.js

```diff
@@ -29,7 +31,10 @@ module.exports = function() {
 	var app = express()
 		.use(allRequests)
 		.use(index)
-		.use(express.static("client"));
+		.use(express.static("client"))
+		.engine("html", expressHandlebars({extname: ".html", helpers: require("../client/js/libs/handlebars")}))
+		.set("view engine", "html")
+		.set("views", path.join(__dirname, "..", "client"));
```

Comment:
The require doesn't happen anymore, so that's easier.

But I'm still using the express rendering because I think that's much simpler than the way we were previously doing it, and will make other things much easier to add on in the future.